### ISSUE TITLE
tabNavigation error

### ISSUE THREAD

**Issue by fangkyi03** [react-navigation/react-navigation#3247]
### Current Behavior
![image](https://user-images.githubusercontent.com/30390014/34701050-045ddc12-f521-11e7-9aa1-bbabe6130010.png)
Because this is a pad project react navigation default that does not support leftTab display

So we modified the source code based on tabCompontTop to implement a tabComponent to display the left content

Because this is a private project that can't be published in the code, it can only be written in a rough way

In fact, the question today is related to the question of yesterday.

Https://github.com/react-navigation/react-navigation/issues/3238

Because I want to automatically locate the page in the specified edit box component when I want to implement an edit box component in the click tabBar

This is the use of this

React-native-keyboard-aware-scroll-view

Because this is the essence of a scrollView, the author is afraid that the author has not used the relationship, so here is a direct use of scrollView as an alternative.

directory structure

ScrollView

-----header

-----tabBarCompontTop

------tab1 height:300

------tab2 height:300

--------tab3 height:600 textInputView

Now a situation is that when I load the default tab1, the height of switching to TAB2 page is also limited to 300. Only after I change the height of tab1, the TAB2 page will change.

Because tab is only used to display data, you jump to the tab3 page when you click the Edit button on the page.

There are some textinput components in it because of the react-native-keyboard-aware-scroll-view

The relationship of the component is normally supposed to be directly located when all the component edit boxes get the focus, but now they don't

The question of saying back to yesterday

If I directly use react-navigation's own navigationOpations:header to achieve the head, it can really return to the previous page after launching a navigationActions.back on tab3 page.

But when I changed it to this way, I met two new problems.

1. when I changed header like this, I don't know why the outermost route header has changed the same way.

2. if I change this way, I have no way to use react-native-keyboard-aware-scroll-view again

If I want to implement this component, I can only modify the underlying source code of react-naviation.

### Expected Behavior
There will be a lot of textinput in the tab3 page that I want to do when I click on the textinput in tabView, which can be properly rolled into the edit box for convenient input

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 1.0.0-beta.11
| react-native     |0.44.3
| node             | 8.4.0
| npm | 5.6.0

**Comment by kelset** [comment]
You are using really old versions of both React Native and react-navigation, unless you update to at least the latest react-navigation version there is not much we can do.

**Comment by brentvatne** [comment]
hi @fangkyi03! there's not much that we can do to help you here unless you can narrow down a very specific issue in react-navigation. it's hard for me to pick this problem apart from the description that you've provided :( try to follow the new issue template, hopefully that will help!
